src/DriverManager.php

```diff
@@ -98,15 +100,16 @@ final class DriverManager
      * @var string[]
      */
     private static array $driverSchemeAliases = [
-        'db2'        => 'ibm_db2',
-        'mssql'      => 'pdo_sqlsrv',
-        'mysql'      => 'pdo_mysql',
-        'mysql2'     => 'pdo_mysql', // Amazon RDS, for some weird reason
-        'postgres'   => 'pdo_pgsql',
-        'postgresql' => 'pdo_pgsql',
-        'pgsql'      => 'pdo_pgsql',
-        'sqlite'     => 'pdo_sqlite',
-        'sqlite3'    => 'pdo_sqlite',
+        'db2'         => 'ibm_db2',
+        'ext_sqlite3' => 'sqlite3',
```

Comment:
IMO, we should just wait until `4.0.x` when `sqlite3` becomes available. Defining connection parameters via a URL is not core functionality, and this specific driver isn't either.

Reply:
I agree. I've reverted the changes to the URL parsing.